src/neptune/new/attributes/namespace.py

```diff
@@ -63,6 +68,19 @@ def __len__(self) -> int:
     def __iter__(self) -> Iterator[str]:
         yield from self._attributes.__iter__()
 
+    def extend(
+        self,
+        value: Union[DataTV, Iterable[DataTV]],
```

Comment:
My bad, right.

Reply:
But, I've replaced type with: `value: Union[Any, Iterable[Any]]`.
Lame, but I would use something like `ExtendDictT`.
On the other hand `ExtendDictT` is not explicit enough:
```
ExtendDictT = Union[Collection[Any], Dict[str, "ExtendDictT"]]
```

Because Dict should be replaced with more generic `Dict | Namespace` object.
Please don't focus on types in this PR.

> Should extend work for a single value? Shouldn't there be a separate append method?

No and no. `Handler.assign` delegates call directly to `Handler.extend`

Reply:
> Please don't focus on types in this PR.
Why?

Sorry, I am still confused and your comment is not clear to me. Why extend accept a single value.

Reply:
> Why extend accept a single value.

It accepts `Collection` of single types or nested structure build of `Dicts` or `Namespaces` with `Collection` of simple types as leaves.